### ISSUE TITLE
Windows build failed tests

### ISSUE THREAD

**Issue by AndreaLanfranchi** [torquem-ch/silkworm#10]
Total Test time (real) =  25.73 sec
The following tests FAILED:
	  1 - Storage change (Exit code 0xc0000409)
	  6 - Smart contract with storage (Exit code 0xc0000409)
	  7 - Maximum call depth (Exit code 0xc0000409)
	  8 - DELEGATECALL (Exit code 0xc0000409)
	 14 - BN_MUL (Failed)
	 15 - SNARKV (Failed)
	 17 - No refund on error (Exit code 0xc0000409)
	 18 - Self-destruct (Exit code 0xc0000409)
	 20 - Empty suicide beneficiary (Exit code 0xc0000409)

Exit code 0xc0000409 is vector index out of range

**Comment by AndreaLanfranchi** [comment]
Closed by #11

**Comment by AndreaLanfranchi** [comment]
Woops seems like I closed too early.
After merge with #11 I still get this output:

```
Test project C:/Users/Andrea.ISIWEB/CMakeBuilds/silkworm/build/x64-Debug
      Start  1: Storage change
 1/34 Test  #1: Storage change ..........................   Passed    0.17 sec
      Start  2: History index search
 2/34 Test  #2: History index search ....................   Passed    0.16 sec
      Start  3: LMDB
 3/34 Test  #3: LMDB ....................................   Passed    0.17 sec
      Start  4: Create address
 4/34 Test  #4: Create address ..........................   Passed    0.16 sec
      Start  5: Value transfer
 5/34 Test  #5: Value transfer ..........................   Passed    0.16 sec
      Start  6: Smart contract with storage
 6/34 Test  #6: Smart contract with storage .............   Passed    0.16 sec
      Start  7: Maximum call depth
 7/34 Test  #7: Maximum call depth ......................   Passed    0.48 sec
      Start  8: DELEGATECALL
 8/34 Test  #8: DELEGATECALL ............................   Passed    0.16 sec
      Start  9: CREATE should only return on failure
 9/34 Test  #9: CREATE should only return on failure ....   Passed    0.16 sec
      Start 10: Ecrecover
10/34 Test #10: Ecrecover ...............................   Passed    0.16 sec
      Start 11: SHA256
11/34 Test #11: SHA256 ..................................   Passed    0.16 sec
      Start 12: RIPEMD160
12/34 Test #12: RIPEMD160 ...............................   Passed    0.16 sec
      Start 13: EXPMOD
13/34 Test #13: EXPMOD ..................................***Failed   14.07 sec
      Start 14: BN_ADD
14/34 Test #14: BN_ADD ..................................   Passed    0.16 sec
      Start 15: BN_MUL
15/34 Test #15: BN_MUL ..................................***Failed    0.16 sec
      Start 16: SNARKV
16/34 Test #16: SNARKV ..................................***Failed    0.18 sec
      Start 17: Zero gas price
17/34 Test #17: Zero gas price ..........................   Passed    0.16 sec
      Start 18: No refund on error
18/34 Test #18: No refund on error ......................   Passed    0.16 sec
      Start 19: Self-destruct
19/34 Test #19: Self-destruct ...........................   Passed    0.16 sec
      Start 20: Out of Gas during account re-creation
20/34 Test #20: Out of Gas during account re-creation ...   Passed    0.17 sec
      Start 21: Empty suicide beneficiary
21/34 Test #21: Empty suicide beneficiary ...............   Passed    0.16 sec
      Start 22: RLP decoding
22/34 Test #22: RLP decoding ............................   Passed    0.16 sec
      Start 23: RLP encoding
23/34 Test #23: RLP encoding ............................   Passed    0.16 sec
      Start 24: HashBuilder
24/34 Test #24: HashBuilder .............................   Passed    0.16 sec
      Start 25: Hardcoded root hash
25/34 Test #25: Hardcoded root hash .....................   Passed    0.16 sec
      Start 26: Account RLP
26/34 Test #26: Account RLP .............................   Passed    0.16 sec
      Start 27: Decode account from storage
27/34 Test #27: Decode account from storage .............   Passed    0.16 sec
      Start 28: BlockBody RLP
28/34 Test #28: BlockBody RLP ...........................   Passed    0.16 sec
      Start 29: Hardcoded Bloom
29/34 Test #29: Hardcoded Bloom .........................   Passed    0.16 sec
      Start 30: Transaction RLP
30/34 Test #30: Transaction RLP .........................   Passed    0.16 sec
      Start 31: Padding
31/34 Test #31: Padding .................................   Passed    0.16 sec
      Start 32: Zeroless view
32/34 Test #32: Zeroless view ...........................   Passed    0.16 sec
      Start 33: to_hash
33/34 Test #33: to_hash .................................   Passed    0.16 sec
      Start 34: DB encode timestamp
34/34 Test #34: DB encode timestamp .....................   Passed    0.16 sec
91% tests passed, 3 tests failed out of 34
Total Test time (real) =  19.86 sec
The following tests FAILED:
	 13 - EXPMOD (Failed)
	 15 - BN_MUL (Failed)
	 16 - SNARKV (Failed)
Errors while running CTest
```
